tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -171,16 +195,17 @@ ROOT::Experimental::Detail::RPageSinkDaos::CommitPageImpl(ColumnHandle_t columnH
    return CommitSealedPageImpl(columnHandle.fId, sealedPage);
 }
 
-
 ROOT::Experimental::RNTupleLocator
-ROOT::Experimental::Detail::RPageSinkDaos::CommitSealedPageImpl(
-   DescriptorId_t /*columnId*/, const RPageStorage::RSealedPage &sealedPage)
+ROOT::Experimental::Detail::RPageSinkDaos::CommitSealedPageImpl(DescriptorId_t columnId,
+                                                                const RPageStorage::RSealedPage &sealedPage)
 {
    auto offsetData = fOid.fetch_add(1);
+   DescriptorId_t clusterId = fDescriptorBuilder.GetDescriptor().GetNClusters();
+
    {
       RNTupleAtomicTimer timer(fCounters->fTimeWallWrite, fCounters->fTimeCpuWrite);
-      fDaosContainer->WriteSingleAkey(sealedPage.fBuffer, sealedPage.fSize,
-                                      {offsetData, 0}, kDistributionKey, kAttributeKey);
+      RDaosKey daosKey = GetPageDaosKey<kOClusterDColumn>(clusterId, columnId, offsetData);
```

Comment:
In its current form, to switch to a different mapping we have to change the template parameter in all the calls to `GetPageDaosKey()`.  It would help if we add a `constexpr` definition near lines 56-65 as in:
```c++
constexpr EDaosMapping kDefaultDaosMapping = kOClusterDColumn;
```